example/private-stablecoin/src/index.ts

```diff
@@ -0,0 +1,426 @@
+import PaladinClient, {
+  PaladinVerifier,
+  TransactionType,
+  ZetoFactory,
+  algorithmZetoSnarkBJJ,
+  IDEN3_PUBKEY_BABYJUBJUB_COMPRESSED_0X,
+} from "@lfdecentralizedtrust-labs/paladin-sdk";
+import { checkDeploy, checkReceipt } from "paladin-example-common";
+import erc20Abi from "./abis/SampleERC20.json";
+import kycAbi from "./abis/IZetoKyc.json";
+import { buildBabyjub } from "circomlibjs";
+
+const logger = console;
+
+const paladin1 = new PaladinClient({
+  url: "http://127.0.0.1:31548",
+});
+const paladin2 = new PaladinClient({
+  url: "http://127.0.0.1:31648",
+});
+const paladin3 = new PaladinClient({
+  url: "http://127.0.0.1:31748",
+});
+
+async function getBabyjubPublicKey(verifier: PaladinVerifier): Promise<string[]> {
```

Comment:
Is there a simpler way to do this, or a way to make Paladin/Zeto do it for us? Feels like this utility will be necessary for anyone using this variant of Zeto.

Reply:
Agree, the best approach I think is adding a new key spec for the uncompressed form, in addition to the compressed form, to be returned from the `Resolve()` RPC call. I'll work on adding that.